tools/android_lint/bin/main.dart

```diff
@@ -0,0 +1,88 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart' as path;
+import 'package:process/process.dart';
+
+Future<void> main(List<String> args) async {
+  final ArgParser argParser = ArgParser();
+  argParser.addOption(
+    'in',
+    help: 'The path to `engine/src`.',
+    defaultsTo: path.join(path.current),
```

Comment:
You could have it find the default based on the location of the script, using `Platform.script`:
```dart
final String flutterRoot = path.dirname(path.dirname(path.dirname(path.fromUri(Platform.script))));
```
This makes it possible to run it from any dir without needing to specify `--in` (although you should keep that as an option).

Reply:
Done with a little twist to make sure it's still relative so that project.xml comes out ok on other systems